{{cookiecutter.repo_name}}/.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,60 @@
+## For the Submitter
```

Comment:
I think it's a good point that keeping the template trim will reduce the overall overhead. That said, I would prefer to have it as a checklist that at least asks you to think about each of these things, every time. Let me set up another PR that tries to balance those concerns a bit more.